tests/helper_test_functions.py

```diff
@@ -0,0 +1,140 @@
+"""Helper functions for test."""
+from _pytest.fixtures import FixtureRequest
+import pytest
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.remote.webelement import WebElement
+from selenium.webdriver.support import expected_conditions as ec
+from selenium.webdriver.support.wait import WebDriverWait
+
+from . import settings
+
+TIMEOUT_BUTTON = 2
+TIMEOUT_TEXTFIELD = 2
+
+ID_USERNAME_ELEMENT = "username_textfield"
+ID_PASSWORD_ELEMENT = "password_textfield"
+
+
+# The name used for the username text field
+# The id used for the login button
+ID_LOGIN_BUTTON_ELEMENT = "login_button"
+
+# The id used for the logout button in the home page
+ID_LOGOUT_ELEMENT = "logout_element"
+
+
+@pytest.fixture(scope="class")
```

Comment:
Why is scope set to `"class"`? See https://docs.pytest.org/en/6.2.x/reference.html#pytest-fixture

Reply:
The scope makes it so that a new driver would be opened for different classes. So that you have more control over the driver when you write the test in your own class because you would get your own driver. I think it is better than function scope because it is faster to run the test because you do not need to reopen the driver for every test that you are doing.